lib/recurly/coupon.php

```diff
@@ -56,6 +68,17 @@ public static function deleteCoupon($couponCode, $client = null) {
     return Recurly_Base::_delete(Recurly_Coupon::uriForCoupon($couponCode), $client);
   }
 
+  protected function createUpdateXML() {
```

Comment:
I was having a hard time thinking of anything, hence the call for the comment. `xmlForUpdate` is the least bad thing I can think of.